scripts/packing_similarity_dendrogram/packing_similarity_dendogram.py

```diff
@@ -282,12 +287,14 @@ def main(input_file, matrix_file, n_ps_mols, output_ps_results, conf_threshold,
                 os.makedirs(overlay_folder)
 
         for i in range(0, structure_size):
-            refcodes.append(str(structure_reader[i].identifier))
+            refcodes.append(str(i+1))
+#            refcodes.append(str(structure_reader[i].identifier))#IJS 06/09/22 why are refcodes read in twice?
 
         for i in range(0, structure_size):
             entry_i = structure_reader[i]
             crystal_i = entry_i.crystal
-            refcodes.append(str(structure_reader[i].identifier))
+            refcodes.append(str(i+1))
+#            refcodes.append(str(structure_reader[i].identifier))
```

Comment:
I don't understand why this for loop is used twice 🤔

Reply:
neither do I!